src/platform/common/platform/fileSystem.ts

```diff
@@ -30,16 +30,7 @@ export class FileSystem implements IFileSystem {
 
     // API based on VS Code fs API
     arePathsSame(path1: vscode.Uri, path2: vscode.Uri): boolean {
-        if (path1.scheme === 'file' && path1.scheme === path2.scheme) {
-            // eslint-disable-next-line local-rules/dont-use-fspath
-            return this.areLocalPathsSame(path1.fsPath, path2.fsPath);
-        } else {
-            return path1.toString() === path2.toString();
-        }
-    }
-
-    areLocalPathsSame(path1: string, path2: string): boolean {
-        return arePathsSame(path1, path2);
+        return path1.toString() === path2.toString();
```

Comment:
There is a isEqual on URI that should be used for this. It has an ignoreCase option I believe for windows.

Reply:
If you look up areInterpreterPathsSame it's using that isEqual function.

Reply:
do you mean on `vscode.Uri`? I can't find that `isEqual`

We do have a `arePathsSame` in `platform/fileUtils.ts` though